src/drivers/esp32/src/esp_slip.c

```diff
@@ -42,3 +44,32 @@ static uint8_t generateChecksum(uint8_t *sendBuffer, esp_slip_send_packet *sende
   }
   return checksum;
 }
+
+static void assembleBuffer(uint8_t *sendBuffer, esp_slip_send_packet *senderPacket)
+{
+  sendSize = senderPacket->dataSize + ESP_OVERHEAD_LEN + 2;
```

Comment:
Why + 2?

Reply:
It's for the start and end bytes. I'll add a comment to explain it.

Reply:
I added a comment to explain it's for the start and end byte of the SLIP packet